src/containers/Transactions/transaction.scss

```diff
@@ -57,18 +57,30 @@
       }
     }
 
-    .hash {
+    .txid {
+      .title {
+        @include semibold;
+
+        margin-right: 2px;
```

Comment:
Would prefer to use a real space instead of a margin.  That way it scales with font-size changes better.